docs/config/header.html

```diff
@@ -27,7 +27,26 @@
   <script type="text/javascript" src="$relpath^dynsections.js"></script>
   $treeview
   $search
-  $mathjax
+  <script type="text/x-mathjax-config">
```

Comment:
Only the latter is an option.

Reply:
Fine with me, but why? :)

Reply:
Because Doxygen otherwise will try to load multiple MathJax scripts and I expect that to cause problems. If you don't specify `MATHJAX_RELPATH` it defaults to some MathJax 2 version and so we would then have a load for MathJax 3 in the `mathjax_helper.js` and a load for MathJax 2 from Doxygen. By having Doxygen use the MathJax.js file we get all the config into one file with no weird loading different versions. The first option you suggested might be possible, but definitely would require more care and would require more things to be synced up. Does that answer the question? :)